dts/bindings/bluetooth/bt-hci.yaml

```
@@ -20,6 +20,7 @@ properties:
       - "spi"
       - "i2c"
       - "ipm"
```

Would this be useful?

```suggestion
      - "ipm"  # Deprecated alias for "ipc"
```

---

Done.